assets/inference/environments/minimal-ubuntu20.04-py39-cpu-inference/environment.yaml

```diff
@@ -0,0 +1,12 @@
+image:
+  name: azureml/curated/minimal-py39-inference
```

Comment:
Change image name to minimal-ubuntu20.04-py39-cpu-inference? This is the name published in MCR.